R/loo_subsample.R

```diff
@@ -455,7 +455,7 @@ nobs.psis_loo_ss <- function(object, ...){
 #' @param api the choices available in the loo API or all possible choices.
 #' @return a character vector of allowed choices
 loo_approximation_choices <- function(api = TRUE) {
-  lac <- c("plpd", "lpd", "waic", "waic_grad_marginal", "waic_grad", "waic_hess", "none")
+  lac <- c("plpd", "lpd", "waic", "waic_grad_marginal", "waic_grad", "waic_hess", "tis", "none")
```

Comment:
Out of interest: Any reason for not also adding `sis` here?

Reply:
No, it is just that it has not been studied in our paper. I can include it.

Reply:
Now included